indico/modules/categories/templates/display/event_list.html

```diff
@@ -131,6 +131,7 @@ <h4 class="{% if month.is_current %}current-month{% endif %}">
                                         {%- trans %}new{% endtrans -%}
                                     </span>
                                 {% endif %}
+                                {{ template_hook('event-status-flag', event=event) }}
```

Comment:
Since the things here are labels, maybe this name is better? The "flag" part seems to be more what you end up using it for ;)

```suggestion
                                {{ template_hook('event-status-labels', event=event) }}
```

Reply:
Changed!